brave/brave-core
# Adds error flow for corrupted wallet

Resolves https://github.com/brave/brave-browser/issues/2183

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- close browser
- modify ledger_state and delete paymentId
- start browser
- make sure that you get error about corrupted wallet

## Reviewer Checklist:
